src/fetchcode/package.py

```diff
@@ -362,6 +372,123 @@ def get_gnu_data_from_purl(purl):
     )
 
 
+@router.route("pkg:cocoapods/.*")
+def get_cocoapods_data_from_purl(purl):
+    """
+    Generate `Package` object from the `purl` string of cocoapods type
+    """
+    logging.basicConfig(
+        filename=LOG_FILE_LOCATION,
+        level=logging.WARN,
+        format="%(levelname)s - %(message)s",
+        filemode="w",
+    )
+
+    purl = PackageURL.from_string(purl)
+    name = purl.name
+    version = purl.version
+    cocoapods_org_url = f"https://cocoapods.org/pods/{name}"
+    repository_homepage_url = f"https://cocoapods.org/pods/{name}"
```

Comment:
why track two variables with the exact same value?

Reply:
Deleted.